testing/events.go

```diff
@@ -56,6 +57,65 @@ func ParseChannelIDFromEvents(events sdk.Events) (string, error) {
 	return "", fmt.Errorf("channel identifier event attribute not found")
 }
 
+// ParsePacketFromEvents parses events emitted from a MsgRecvPacket and returns the
+// acknowledgement.
+func ParsePacketFromEvents(events sdk.Events) (channeltypes.Packet, error) {
```

Comment:
I'll go back and add this to main later and clean up the tests that use relay packet the hard way (pre constructing the packet)